service/models.py

```diff
@@ -18,10 +18,142 @@ def init_db(app):
     """ Initializes the SQLAlchemy app """
     Customer.init_db(app)
 
-
 class DataValidationError(Exception):
     """ Used for an data validation errors when deserializing """
 
+class Address(db.Model):
+    """
+    Class that represents a Address
+    """
+
+    ################
+    # Address Schema
+    ################
+
+    logger = logging.getLogger('flask.app')
```

Comment:
we already have a global logger in line 11, any other use case for this?

Reply:
Removed the local Address logger.